components/event-publisher-proxy/pkg/metrics/collector.go

```diff
@@ -9,21 +9,21 @@ import (
 
 const (
 	// Errors name of the errors metric
-	Errors = "event_publish_to_messaging_server_errors_total"
+	Errors = "eventing_epp_errors_total"
```

Comment:
can we follow similar variable naming conventions across the two projects? 
I would recommend using `ErrorsKey` and `ErrorsHelp` as variable names here like we do in the other collector.go file

Reply:
done